Developer/macOS/ExtendingAudioKit/README.md

```diff
@@ -1,53 +1,74 @@
+
+At this point you should build your project to make sure everything is OK. This should happen quite quickly.
+
+### Option 2: Add AudioKit as a sub-project
+
 In the Mac Finder, locate the *AudioKit For macOS.xcodeproj* project and drag it into the your own Xcode project:
 
 * In the leftmost pane, make sure the Project Navigator is showing
 * Drag the AudioKit project file and drop it directly under the top line which represents your new project.
 
-In the Xcode Project page, click on the General tab, and scroll to the Embedded Binaries section. Click the + button — a “Choose items to add” dialog will pop up, in which you’ll see two copies of both *AudioKit.framework* and *AudioKitUI.framework*. Choose both frameworks (which of the duplicate copies you choose may not matter; I always choose the ones nearest the bottom) and click Add.
+In the Xcode Project page, click on the General tab, and scroll to the Linked Frameworks and Libraries section. Click the + button — a “Choose frameworks and libraries to add” dialog will pop up, in which you’ll see *AudioKit.framework* and *AudioKitUI.framework*, right at the top under "Workspace". Select both frameworks and click Add.
 
 At this point you should Build your project to make sure everything is OK. This will take quite some time, as the whole of AudioKit must be compiled, but this will only need to happen once.
 
-Create a bridging header file for your project:
+### Create a bridging header file for your project:
 
-* In the Xcode Project Navigator, right-click your project icon, select Header, name your file as *[your project name]-Bridging-Header.h*, and save it into the same folder as your project file.
+* In the Xcode Project Navigator, right-click your project icon and choose "New File...", select Header File, name your file as *[your project name]-Bridging-Header.h*, and save it into the same folder as your project file.
 * The file will be created with the old #ifndef … #endif syntax. Replace with #pragma once.
 
 Back in your project’s Build Settings, locate the Objective-C Bridging Header item (search for “bridging”). Set it to *$(SRCROOT)/[your project name]-Bridging-Header.h*.
 
-Next locate the Header Search Paths item (search for “search”), and set it to point to the *AudioKit/AudioKit/Common/Internals/CoreAudio* folder:
+### Prepare to use AudioKit core header files
+Before you can build your own extensions to AudioKit, you need to ensure that your Xcode project can make use of some of AudioKit's core header files.
+
+Locate the Header Search Paths item (search for “search”), and set it to point to the *AudioKit/AudioKit/Common/Internals/CoreAudio* folder:
```

Comment:
I don't think this is how I implemented it in the end, I think I pointed to the Headers folder generated in the framework because this would contain the headers that were properly accessible once the framework was used whereas the CoreAudio reference would a) not pick them all up and b) may accidentally see a file that wasn't actually included with AK (unlikely but possible).

Reply:
I'm happy to accept this PR and have another be created as time becomes available.